src/include/catalog/catalog_structs.h

```diff
@@ -127,9 +127,9 @@ struct RelTableSchema : TableSchema {
 
     inline uint32_t getNumProperties() const { return properties.size(); }
 
-    inline uint32_t getNumPropertiesToReadFromCSV() const {
-        // -1 here is to remove the INTERNAL_ID_SUFFIX = "_id" property.
-        return properties.size() - 1;
+    static inline uint32_t getNumNonRelIDProperties() {
```

Comment:
This is not a bug, because i changed the way how we used this function as well. I treated it as a function which returns the numOfGeneratedProperties().
As we discussed, let's rename this function to `getNumUserDefinedProperties`